app/models/agents/gap_detector_agent.rb

```diff
@@ -8,6 +8,11 @@ class GapDetectorAgent < Agent
       The `value_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value of interest. If either
       this value is empty, or no Events are received, during `window_duration_in_days`, an Event will be created with
       a payload of `message`.
+
+      The `stream_path` value is a [JSONPath](http://goessner.net/articles/JsonPath/) to a value in the event that will
```

Comment:
"in receive value_path needs to be present, if stream_path is set as well it needs to have either the same path or be a descendant of value_path, only setting stream_path does not work."

This is not correct.  `value_path` and `stream_path` are completely independent.  `value_path` is only used if it is set.  `stream_path` does work on it own.